src/config/section/role.js

```diff
@@ -55,7 +55,7 @@ export default {
       args: ['name', 'description', 'type'],
       mapping: {
         type: {
-          options: ['Admin', 'DomainAdmin', 'User']
+          options: ['Admin', 'DomainAdmin', 'ResourceAdmin', 'User']
```

Comment:
@davidjumani Remove ResourceAdmin, we deliberately don't want users/admins to use this role type (historic issue, related to cpbm no longer used now)

Reply:
Then shall I remove it while creating a user too ?

Reply:
@davidjumani no leave that in the createAccount form/component for legacy reasons, I think old UI shows the same list of roles as returned by listRoles.